packages/aws-rfdk/lib/core/lib/utils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class Utils {
+  /**
+   * This method compares two version strings
+   *
+   * @param versionFirst
+   * @param versionSecond
+   *
+   * @returns negative value if first version is smaller than second version;
+   * 0 if the versions matches, positive value if first version is smaller
+   * than second version.
+   */
+  public static versionCompare(versionFirst: string, versionSecond: string): number {
+    const regExStripZero = /(\.0+)+$/;
+    const versionArrayFirst = versionFirst.replace(regExStripZero, '').split('.');
+    const versionArraySecond = versionSecond.replace(regExStripZero, '').split('.');
+    const l = Math.min(versionArrayFirst.length, versionArraySecond.length);
+
+    for (let i = 0; i < l; i++) {
+      let diff = parseInt(versionArrayFirst[i], 10) - parseInt(versionArraySecond[i], 10);
```

Comment:
Neat: I guess you should be able to use const here.
```suggestion
      const diff = parseInt(versionArrayFirst[i], 10) - parseInt(versionArraySecond[i], 10);
```

Reply:
updated.